### ISSUE TITLE
different uses and functions of the sequencers

### ISSUE THREAD

**Issue by joopvdl** [MarcBoule/ImpromptuModular#36]
This is not really a request, but a suggestion.
I am using your sequencers a lot, and I am still confused about which one to use when.
At this point, most of them do mostly what I want, but there's often something in another sequencer that I would also need.
For example: Foundry is great when programming, but the sequences cannot be played in a non-predefined order. 
Phrase seq is my current favorite, but I need 3 (or more) of them to build harmonies. (And even more for bass, melodie, etc). 

Anyway, this is not a feature request issue, but I think it would be practical if you made an introduction to the manual about the general philosophy behind each of the sequencers, and the differences between them. What's the main usage you had in mind for this one? Could be the feature I am looking for would not be part of that idea, and I am using the sequencer wrong, so to say. 

I know this is not the place for applauding, but your sequencers are the most musical and practical to use, by far. Only when I want generated melodies or more unpredictability I will turn to  something else, like Squonk or Turing or Random Walk. Or VCV's trigger sequencers.

**Comment by MarcBoule** [comment]
I know what you mean regarding Foundry vs PhraseSeqs, and that is actually why I chose to use a new name for Foundry, to help convey that there are notable differences compared to the PhraseSeqs. For the inability to directly play a sequence, it was a design choice given the multitrack nature of Foundry. If we were to have the previous behavior of the PhraseSeq, when we would move to a give sequence in one track, the same sequence number may not even be defined in another track, so those would play garbage. There is only one edit head that controls editing on all trakcs. I tried it with four editing heads, and it was hell! Also, with the song BEG and END positions that can be easily moved around, we can focus on the part of the song we are working on (even a single seq if we want), but agree that it might take some getting used to. It was also a personal design criteria of mine that this sequencer allow the editing of individual sequences while the song is playing, something not possible in the PS series.

I will think of a kind of feature comparison chart that would show the main differences between all my sequencers, and it would make choosing the right one much easier. So much to do, so little time! :-) 

I very much appreciate the kind words Joop! And the discussion too, feel free to respond in any way to any of this.

**Comment by MarcBoule** [comment]
Foundry was just updated in 0.6.17 to make the SEQ/SONG switch work just like in the PhraseSeqs, so my first paragraph above is now quite outdated :-)

With the table I added in the manual, I hope that the main differences in the sequencers can be quickly seen. I'll close for now, but if ever you want to follow up on something, feel free to comment and I'll re-open.

Thanks for the comments Joop. The new Foundry version (with the SEQ/SONG that works the same as in the PhraseSeqs) has just been released in the plugin manager. Cheers!
